src/Subscriber.ts

```diff
@@ -247,18 +247,18 @@ class SafeSubscriber<T> extends Subscriber<T> {
     }
   }
 
-  private __tryOrUnsub(fn: Function, value?: any): void {
+  private __tryOrUnsub(fn: Function, ...value: Array<any>): void {
     try {
-      fn.call(this._context, value);
+      fn.call(this._context, ...value);
```

Comment:
@kwonoj yeah, but it transpiles down to an Array allocation + apply call, which is 20-25x slower than the equivalent conditional call [in this jsperf](https://jsperf.com/fn-apply-args-vs-cond-call/1) test.